lib/app.js

```diff
@@ -258,10 +259,17 @@ class App {
    * @param  {[type]} root working directory
    * @return {[type]}      map with relativized keys
    */
-  makeKeysRelative(map, root) {
+  makeKeysRelative(root) {
     const newCoverage = {};
-    Object.keys(map).forEach(pathKey => {
-      newCoverage[path.relative(root, pathKey)] = map[pathKey];
+    // NOTE: just a personal preference but it would be much cleaner for linting to
+    //       allow
+    //       const { coverage } = this.coverage;
+    // ref rule: object-curly-newline
+    const {
```

Comment:
Yes! Add that rule if you'd like. Sound's reasonable. Also this rewrite of `makeKeysRelative` is nice.

Reply:
I'll change the rule to `"object-curly-newline": ["error", { "minProperties": 3 }],`